QuantConnect.BybitBrokerage/BybitBrokerage.Messaging.cs

```diff
@@ -179,6 +192,7 @@ private void HandleOrderUpdate(JToken message)
         foreach (var order in orders)
         {
             //We're not interested in order executions here as HandleOrderExecution is taking care of this
+            // TODO: why do we need this method then
```

Comment:
The execution topic only contains order trade executions but no information about the order itself. Without subscribing to this topic and handling the messages we wouldn't get notified about for example order cancel events. 
You can check out the mapping in `BybitBrokerage.cs` - `ConvertOrderStatus`.
We probably don't need to publish all the updates to lean, so it would be great if you could double-check the mapping and exclude the events we don't need.